packages/web/src/press/PressPage.tsx

```diff
@@ -20,29 +20,40 @@ export interface PressArticleFields {
 
 interface Props {
   press: PressArticleFields[]
+  languages: string
 }
 
 class PressPage extends React.PureComponent<I18nProps & Props> {
   static async getInitialProps(context) {
-    let press = []
```

Comment:
can we leave this as defaulting to an array? if something messes up at least then we get an empty array to call array methods on and not `undefined doesnt have function` errors